code/modules/surgery/advanced/brainwashing.dm

```diff
@@ -48,6 +48,9 @@
 	if(!target.mind)
 		to_chat(user, span_warning("[target] doesn't respond to the brainwashing, as if [target.p_they()] lacked a mind..."))
 		return FALSE
+	if(is_banned_from(target.ckey, ROLE_BRAINWASHED))
+		to_chat(user, span_alert("[target] doesn't seem to respond to the brainwashing!"))
+		return FALSE
```

Comment:
Being banned should not confer metaprotection from being brainwashed. This could lead to people requesting a brainwash ban specifically to become immune to brainwashing.

It should instead kick them out of their body and offer them up to ghosts similar to how other conversions are handled.